testing/test_integration.py

```diff
@@ -30,18 +30,10 @@ def fin():
     return server
 
 
-def test_toggle(server):
-    server.add_feature("foo", feature("foo", "jim")['foo'])
-    client = LDClient("apikey", Config(base_uri=server.url, events_uri=server.url))
-    wait_until(lambda: client.toggle("foo", user('xyz'), "blah") == "jim")
-
-
-def test_sse_init(server, stream):
```

Comment:
this is tested by test_toggle, and a bunch of times in the integration harness as well